python-checks/src/test/resources/checks/methodShouldBeStatic.py

```diff
@@ -95,6 +95,10 @@ def fun23(slf): # Noncompliant
         if True:
             raise NotImplementedError
 
+    # raise issue, as the error is unlikely to be a NotImplementedError
```

Comment:
@vilchik-elena Actually not, it is trickier than that, see http://stackoverflow.com/questions/18001721/raise-with-no-argument#18001776